src/test/kotlin/org/jetbrains/kotlinx/jupyter/test/repl/CustomLibraryResolverTests.kt

```diff
@@ -244,7 +244,7 @@ class CustomLibraryResolverTests : AbstractReplTest() {
             %use lib1
             5
         """.trimIndent()
-        repl.execute(code)
+        repl.execute(code = code)
```

Comment:
Is it required change?

Reply:
I was facing a strange bug with "method not found", but now it's gone, so no, it's not required